### PR TITLE
Add a tool to convert migu function log to lambdacloud message

### PR DESCRIPTION
**Pull request by wezhang**
Use it like:
`cat <file> | func-log2message > <lambdacloud_message_file>`

Signed-off-by: Zhang Wei weiz@lambdacloud.com

### CODE REVIEW

**Comment by wezhang** [comment]
fix lambdacloud/lanyun#507

File: bin/etl/migu/func-log2message [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  if (_.endsWith(line, '-')) {
+    buf = line;
+  } else {
+    this.emit('data', buf + line);
+  }
+}
+
+function parseFunction(data) {
+  var regex = /(\d{4}-\d\d-\d\d \d\d:\d\d:\d\d\.\d{3}) .*-\s*(\w+)\s([\w.]*)\(.*\).*\|\|\|(.*)\|\|(.*)/i;
```
Comment (sunyonggang) [inline bin/etl/migu/func-log2message]: 这里的‘(._)  ._   |||’可不可以去掉'.*',原始数据：(String, ClientOrderAndPay)|||，反括号后面直接三个‘｜’

**Comment by sunyonggang** [comment]
shipit!!!
:+1:

File: bin/etl/migu/func-log2message (hunk 1 above)
Comment (wezhang) [inline bin/etl/migu/func-log2message]: good catch!

**Pull request merged**